tests/contrib/operators/test_dataproc_operator.py

```diff
@@ -119,6 +120,7 @@ def setUp(self):
                     project_id=GCP_PROJECT_ID,
                     num_workers=NUM_WORKERS,
                     zone=GCE_ZONE,
+                    autoscaling_policy=SCALING_POLICY,
```

Comment:
What do you think about system tests? Do you have time to pay the technological debt?

Reply:
I think this is not the righ time to add system tests. I would add them later if we have time.